rich/pretty.py

```diff
@@ -133,58 +133,37 @@ def _ipy_display_hook(
 
     console = console or get_console()
     if console.is_jupyter:
-        # Delegate rendering to IPython if the object (and IPython) supports it
-        #  https://ipython.readthedocs.io/en/stable/config/integrating.html#rich-display
-        ipython_repr_methods = [
-            "_repr_html_",
-            "_repr_markdown_",
-            "_repr_json_",
-            "_repr_latex_",
-            "_repr_jpeg_",
-            "_repr_png_",
-            "_repr_svg_",
-            "_repr_mimebundle_",
-        ]
-        for repr_method in ipython_repr_methods:
-            method = getattr(value, repr_method, None)
-            if inspect.ismethod(method):
-                # Calling the method ourselves isn't ideal. The interface for the `_repr_*_` methods
-                #  specifies that if they return None, then they should not be rendered
-                #  by the notebook.
-                try:
-                    repr_result = method()
-                except Exception:
-                    continue  # If the method raises, treat it as if it doesn't exist, try any others
-                if repr_result is not None:
-                    return  # Delegate rendering to IPython
-
         # When in a Jupyter notebook let's avoid the display of some specific classes,
         # as they result in the rendering of useless and noisy lines such as `<Figure size 432x288 with 1 Axes>`.
         # What does this do?
         # --> if the class has "matplotlib.artist.Artist" in its hierarchy for example, we don't render it.
         if is_object_one_of_types(value, JUPYTER_CLASSES_TO_NOT_RENDER):
```

Comment:
Having fixed the duplicate output, I'm not sure if this behavior is desired anymore - this was caused because matplotlib formatters were registered via [formatter.for_type](https://github.com/ipython/ipython/blob/9663a9adc4c87490f1dc59c8b6f32cdfd0c5094a/IPython/core/pylabtools.py#L284), so the check above didn't catch that a formatter was registered.

It would make sense to me to either remove this altogether, or `return repr(value)`, because as it is, if inline backend is not enabled, displaying or returning a figure will show _nothing_, rather than the simple repr of the object.

Reply:
I'm happy to remove that. It was a clearly a hack anyway.